### ISSUE TITLE
Create plan for non-labs program clubs

### ISSUE THREAD

**Issue by jonleung** [hackedu/hackedu#52]
For the clubs that are not in the labs program:

- what will our interaction with them be?
- if they reach out for help, what will happen?

I scheduled a lunch meeting to discuss this tomorrow (Tuesday August 17th)

[cc @hellyeah, @zachlatta, @MaxWofford, @andreadeng]

**Comment by MaxWofford** [comment]
This is a duplicate of #51

**Comment by MaxWofford** [comment]
@jonleung What were the outcomes of the meeting?

**Comment by hellyeah** [comment]
Something like this:
1. We want to focus almost solely on labs clubs so we can deliver a tested and true club-in-a-box by the start of 2016. Devoting time to non-labs clubs takes away from time we could spend w/ labs clubs.
2. Non-labs are important bc we want to continue to grow.
3. We should accept clubs that apply and add them to the hackEDU community.
4. We should keep close contact with top non-labs clubs to learn what works for them and try it in labs.

I'm following this up w/ #58

**Comment by MaxWofford** [comment]
@hellyeah Thanks for posting that!

**Comment by jonleung** [comment]
@hellyeah, completely agree! Also, I believe we discussed the following during the meeting:

Two ideas for how we could make the Labs clubs feel special are:

1. Giving them a lot of personal attention and help
2. Highlighting the clubs on social media as well as the media consumed by the general public

The more desirable being a hackEDU Lab Director is in the fall, the more desirable it will be to be a hackEDU club leader this spring.

/cc @MaxWofford, @hellyeah, @zachlatta

**Comment by hellyeah** [comment]
Submitting this as a PR shortly: https://gist.github.com/hellyeah/cf744f491cff2c88c2ff

**Comment by zachlatta** [comment]
Has this been submitted as a PR?

On Fri, Aug 28, 2015 at 3:54 PM, David Fontenot <notifications@github.com>
wrote:

> Submitting this as a PR shortly:
> https://gist.github.com/hellyeah/cf744f491cff2c88c2ff
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackedu/hackedu/issues/52#issuecomment-135905989>.
>



-- 
Zach Latta
hackEDU | zach@hackedu.us

**Comment by hellyeah** [comment]
Waiting on review+merge.
